kge/util/dump.py

```diff
@@ -113,83 +113,167 @@ def _add_dump_trace_parser(subparsers_dump):
     parser_dump_trace = subparsers_dump.add_parser(
         "trace",
         help=(
-            "Process and dump trace to stdout and/or csv. The trace will be processed "
-            "backwards, starting with a specified job_id."
+            "Dump trace to csv (default) and/or stdout. The tracefile will be processed"
+            " backwards starting from the last entry. Further options allow to start"
+            " processing from a particular checkpoint or job_id or epoch_number."
         ),
     )
-
     parser_dump_trace.add_argument(
         "source",
         help="A path to either a checkpoint or a job folder.",
         nargs="?",
         default=".",
     )
-
+    parser_dump_trace.add_argument(
+        "--train",
+        action="store_const",
+        const=True,
+        default=False,
+        help=(
+            "Include entries from training jobs (Note: if none of --train --valid"
+            " --test is set, this will activate all of them jointly)."
```

Comment:
Confusing: write in parens "(enabled when none of --train, --valid, or --test is specified)"